README.md

```diff
@@ -1,56 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Metrics Webapp
 
-> Description the project.
+> A mobile web application to check a list of metrics about Financial stock data.
 
+https://user-images.githubusercontent.com/50138343/158038577-1e01372c-7a8c-4bfc-9007-867cb979795c.mp4
```

Comment:
- [ ] I cannot seem to hear the sound of the video you recorded. Make sure the video has met all the requirements mentioned below: 

- The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.


> **Note:** It's recommended to use [Loom](https://www.loom.com/) to record to video but You can use any tool.